docs/Tutorials/VisualizationWithPython.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualization with Python {#tutorial_vis_python}\n",
+    "\n",
+    "This tutorial shows you how to load 3D volume data in Python and visualize it\n",
+    "using `matplotlib`.\n",
+    "\n",
+    "First, you need access to the `spectre` Python bindings. A simple way to do this\n",
+    "is to run a Jupyter server from the build directory:\n",
```

Comment:
Maybe add a link to the `python.md` tutorial you just edited since that has detailed instructions on how to load the spectre module